.travis.yml

```diff
@@ -3,25 +3,14 @@ sudo: false
 language: node_js
 
 node_js:
-  - "0.8"
```

Comment:
Ah ok. I was afraid of that.

I'm not big on supporting Node versions that are out of support range.
Thanks though.